### PR TITLE
new PR

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

Please include the following:

- A meaningful PR title [guidelines](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#use-imperative-mood-in-your-commit-message-subject)
- A summary of the change
- Which issue is fixed (if there is one)
- Relevant motivation and context

## How can this be tested?

Please include some guiding steps on how to test this change during a review.

- What environment is necessary for the change to be noticeable ?
- What needs to be enabled/applied for the change to be noticeable ?

## Checklist

- [ ] Unit tests have been updated/added
- [ ] PR is labeled accordingly with a single label
- [ ] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: pkg/util/kubeobjects/object_test.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+func TestKey(t *testing.T) {
+	type args struct {
+		object client.Object
+	}
+	tests := []struct {
+		name string
+		args args
+		want client.ObjectKey
+	}{
+		{name: "tttttt", args: args{object: nil}, want: client.ObjectKey{}},
```
Comment (0sewa0) [inline pkg/util/kubeobjects/object_test.go:18]:
```suggestion
		{name: "handle nil value", args: args{object: nil}, want: client.ObjectKey{}},
```

I don't really understand the test, but it kinda looks like a nil check test, would this name not be better?

**Review by 0sewa0 — CHANGES_REQUESTED** [review]

**Review by 0sewa0 — COMMENTED** [review]
This is a test

File: pkg/util/kubeobjects/object_test.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package kubeobjects
```
Comment (0sewa0) [inline pkg/util/kubeobjects/object_test.go:1]: This is a test

File: pkg/util/kubeobjects/object_test.go (hunk 1 above)
Comment (andriisoldatenko) [reply pkg/util/kubeobjects/object_test.go]: fixed in f984e317

File: pkg/util/kubeobjects/object_test.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"testing"
+)
+
+func TestKey(t *testing.T) {
+	type args struct {
+		object client.Object
+	}
+	tests := []struct {
+		name string
```
Comment (0sewa0) [inline pkg/util/kubeobjects/object_test.go:14]:
```suggestion
		title string
```

File: pkg/util/kubeobjects/object_test.go (hunk 2 above)
Comment (andriisoldatenko) [reply pkg/util/kubeobjects/object_test.go:1]: reolied.

File: pkg/util/kubeobjects/object_test.go (hunk 3 above)
Comment (andriisoldatenko) [reply pkg/util/kubeobjects/object_test.go:14]: ldkfajsdlfkjalkdsfj

**Review by 0sewa0 — APPROVED** [review]

**Pull request closed without merging**